server/src/ai/routes.ts

```diff
@@ -0,0 +1,31 @@
+import { Router } from 'express'
+import { minReviewsCosting } from './functions'
+
+const router = Router()
+
+/** Reachable at POST /ai/costing 
+ * @body minimum number of reviews a course needs to be included in data
+ * returns values used for OpenAI model costing consideration
+*/
+router.post('/costing', async (req, res) => {
+  const min = req.body.min;
+
+  try {
+    const costing = await minReviewsCosting(min);
+    if (minReviewsCosting === null) {
+      return res.status(400).json({
+        error: `No reviews found`,
+      });
+    }
+
+    return res.status(200).json({
+      message: 'Retrieved all pending reviews',
+      result: costing,
+    });
+  } catch (err) {
+    return res.status(500).json({ error: `Internal Server Error: ${err}` });
+  }
```

Comment:
I would move around some of these status responses to keep it consistent with the rest of the codebase
```suggestion
  try {
    const costing = await minReviewsCosting(min);
    if (!costing) {
     return res.status(500).json({ error: `Internal Server Error` });
    }

    return res.status(200).json({
      message: `Retrieved all pending reviews for a minimum value of ${min}`,
      result: costing,
    });
  } catch (err) {
    return res
        .status(400)
        .json({ error: `No reviews found` });
  }
```